### PR TITLE
feat: add readme's for catalog

### PR DESCRIPTION
**Pull request by srodenhuis**
Add's readme to all otomi quick start Helm charts to show in the info tab in the Catalog

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
 
 ## otomi-quickstart-k8s-deployment-otel
 
-The `otomi-quickstart-k8s-deployment-otel` chart can be used to create a Kubernetes `Deployment` with Open Telemetry `instrumentation`.
+The `otomi-quickstart-k8s-deployment-otel` Helm chart can be used to create a Kubernetes `Deployment` (to deploy a single image), a `Service`, a `ServiceAccount`, an `OpenTelemetryCollector` and an `Instrumentation`. Optionally a `HorizontalPodAutoscaler`, a Prometheus `ServiceMonitor` and a `Configmap` can be created.
 
 ## otomi-quickstart-k8s-deployments-canary
 
-The `otomi-quickstart-k8s-deployments-canary` chart can be used to create 2 Kubernetes `Deployments` to deploy 2 versions of an image and use the `Traffic Control` feature in Otomi to do canary or blue/green deployments.
+The `otomi-quickstart-k8s-deployment-canary` Helm chart can be used to create 2 Kubernetes `Deployments` (to deploy 2 versions of an image), a `Service` and a `ServiceAccount`. Optionally a `HorizontalPodAutoscaler`, a Prometheus `ServiceMonitor` and a `Configmap` (for each version) can be created.
```
Comment (ferruhcihan) [inline README.md]: Should we change the folder name?
If we change the folder name (without s in deployments), we also need to change the rbac.yaml file content.

File: otomi-quickstart-k8s-deployment-otel/values.yaml [hunk 2]
```diff
@@ -20,7 +20,7 @@ env: []
   # - name: DB_PASSWORD
   #   valueFrom:
   #     secretKeyRef:
-  #       name: cluster-openforms-superuser
+  #       name: ruser
```
Comment (ferruhcihan) [inline otomi-quickstart-k8s-deployment-otel/values.yaml]: It might be user ?

**Review by ferruhcihan — CHANGES_REQUESTED** [review]
The file icons/deploy.png appears to be empty.

File: README.md (hunk 1 above)
Comment (srodenhuis) [reply README.md]: fixed

File: otomi-quickstart-k8s-deployment-otel/values.yaml (hunk 2 above)
Comment (srodenhuis) [reply otomi-quickstart-k8s-deployment-otel/values.yaml]: fixed

**Review by ferruhcihan — APPROVED** [review]
LGTM

**Pull request merged**